### PR TITLE
Gens restructuring

### PR DESCRIPTION
**Pull request by Uzay-G**
- making search happen in an explicit search bar
- making a dual view with gen editor // search interface
- need to iron out design details probably (spacing?)

image

### CODE REVIEW

File: front/pages/w/[wId]/u/gens/index.tsx [hunk 1]
```diff
@@ -1625,17 +1548,126 @@ export default function AppGens({
 
                 <div className="mb-4 mt-2 flex flex-row flex-wrap items-center text-xs font-normal"></div>
               </div>
+              <div className="mt-24 w-1/3">
```
Comment (spolu) [inline front/pages/w/[wId]/u/gens/index.tsx:1551]: The Retrieval should be aligned with the textarea not below it or even aligned at the top?

File: front/pages/w/[wId]/u/gens/index.tsx [hunk 2]
```diff
@@ -1625,17 +1548,126 @@ export default function AppGens({
...
+                <h2 className="text-lg font-bold">Retrieval</h2>
+                <p>
+                  Retrieve documents and ideas that can help you with your Gen.
```
Comment (spolu) [inline front/pages/w/[wId]/u/gens/index.tsx:1554]: `and ideas`  what do you mean by ideas?

**Review by spolu — COMMENTED** [review]
A few questions

File: front/pages/w/[wId]/u/gens/index.tsx [hunk 3]
```diff
@@ -1625,17 +1548,126 @@ export default function AppGens({
...
+                          onClick={handleInferTimeRange}
+                        />
+                      )}
+                    </div>
+                    <GensTimeRangePicker
+                      timeRange={timeRange}
+                      onTimeRangeUpdate={setTimeRange}
+                    />
+                  </div>
+                  <div className="flex flex-row items-center space-x-2 leading-8">
```
Comment (spolu) [inline front/pages/w/[wId]/u/gens/index.tsx:1647]: This won't work aligned on the same line if you click on the time picker? Let's keep them one per line for now?

**Review by spolu — COMMENTED** [review]
Another one :)

**Comment by Uzay-G** [comment]
Updates: 
image

**Comment by Uzay-G** [comment]
I'm sort of blocked on this to make progress re loom/other stuff, so if you don't have time right now I can merge and then modify after?

**Comment by Uzay-G** [comment]
Merging, please make comments when you can :)

**Pull request merged**